seodongheee2/controllers/postController.js

```diff
@@ -0,0 +1,29 @@
+const postService = require('../services/postServices');
+
+const createPost = async (req, res) => {
+  try {
+    const { title, content,userId,postingImageUrl} = req.body;
```

Comment:
가독성을 위해서 쉼표 이후 한 칸 씩 띄어주세요!

```suggestion
    const { title, content, userId, postingImageUrl} = req.body;
```